charts/hub-agent/README.md

```diff
@@ -19,17 +19,24 @@ _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documen
+| namespace                       | Namespace that this Helm chart is installed on                                                                                                               | `fleet-system`                                   |
+| serviceAccount.create           | Whether to create service account                                                                                                                            | `true`                                           |
+| serviceAccount.name             | Service account name                                                                                                                                         | `hub-agent-sa`                                   |
+| resources                       | The resource request/limits for the container image                                                                                                          | limits: 500m CPU, 1Gi, requests: 100m CPU, 128Mi |
+| affinity                        | The node affinity to use for hubagent pod                                                                                                                    | `{}`                                             |
+| tolerations                     | The tolerations to use for hubagent pod                                                                                                                      | `[]`                                             |
+| logVerbosity                    | Log level. Uses V logs (klog)                                                                                                                                | `5`                                              |
+| enableV1Alpha1APIs              | If set, the agents will watch for the v1alpha1 APIs.                                                                                                         | `false`                                          |
+| enableV1Beta1APIs               | If set, the agents will watch for the v1beta1 APIs.                                                                                                          | `true`                                           |
+| hubAPIQPS                       | QPS to use while talking with fleet-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.   | `20.0`                                           |
+| hubAPIBurst                     | Burst to use while talking with fleet-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags. | `100`                                            |
+| concurrentClusterPlacementSyncs | The number of clusterResourcePlacement to run concurrently.                                                                                                  | `10`                                             |
```

Comment:
```suggestion
| concurrentClusterPlacementSyncs | The max number of clusterResourcePlacement to run concurrently this fleet supports.                                                                                                  | `10`                                             |
```

Reply:
given the name is not what it looks like, can we change the name?

Reply:
do you want to change this in a following up PR? I mean change the "concurrentClusterPlacementSyncs" to reflect what it actually it

Reply:
I'll change it on here.